src/hotspot/share/runtime/thread.hpp

```diff
@@ -1086,7 +1074,7 @@ class JavaThread: public Thread {
 
   // last_Java_sp
   bool has_last_Java_frame() const               { return _anchor.has_last_Java_frame(); }
-  intptr_t* last_Java_sp() const                 { return _anchor.last_Java_sp(); }
+  intptr_t* last_Java_sp()   const               { return _anchor.last_Java_sp(); }
```

Comment:
Why the whitespace change?

Reply:
I try to align with above "const". I will format this patch better next time.